ocaml/xapi/message_forwarding.ml

```diff
@@ -2501,6 +2501,10 @@ functor
             let snapshot = Db.VM.get_record ~__context ~self:vm in
             reserve_memory_for_vm ~__context ~vm ~host ~snapshot
               ~host_op:`vm_migrate (fun () ->
+                Db.VM.get_VGPUs ~__context ~self:vm
+                |> List.iter (fun _ ->
```

Comment:
If a VM has multiple vGPUs, should we not still count it as a single vGPU migration?